### PR TITLE
License and conduct

### PR DESCRIPTION
**Pull request by ghost**
Adds appropriate license and code of conduct

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The following license is modified from the MIT license.
+
+ANTI-FASCIST LICENSE:
+
+The following conditions must be met by any person obtaining a copy of this
+software:
+
+- You MAY NOT be a fascist.
+- You MUST not financially support fascists.
```
Comment (PopNotSoda) [inline LICENSE.md:11]: Consider capitalization on `not`. Same for next line

File: CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (aguynamedjoetoo) [inline CONDUCT.md:58]: idk if we still have the mod email @rattusrattus ?

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- You MUST not publicly voice support for fascists.
+
+"Fascist" can be understood as any entity which supports radical authoritarian
+nationalism. For example: Donald Trump is a fascist; if you donated to his
```
Comment (aguynamedjoetoo) [inline LICENSE.md:15]: lol -- I don't mind adding a license, let's just go with regular MIT pls.

10/10 high effort post

File: CONDUCT.md (hunk 2 above)
Comment (rattusrattus) [reply CONDUCT.md:58]: Perhaps we use the twitter account as a point of contact?  Seems like the drama would be from there anyway.

https://twitter.com/rSeattleWA/

And all mods already have access to it today (in theory).

**Pull request closed without merging**